packages/dataviews/src/types.ts

```diff
@@ -158,6 +158,11 @@ export type Field< Item > = {
 	 * Defaults to `item[ field.id ]`.
 	 */
 	getValue?: ( args: { item: Item } ) => any;
+
+	/**
+	 * Whether the action can be used as a bulk action.
+	 */
+	supportsBulk?: boolean;
```

Comment:
Should we be more explicit? `supportsBulkEditing` or something like that. We may have bulk actions or something else.